src/lib/userStorage/UserStorageClass.js

```diff
@@ -894,7 +895,14 @@ export class UserStorage {
 
     data.value = get(receiptEvent, 'value') || get(receiptEvent, 'amount') || amount
 
-    const fromGDUbi = this.wallet.getUBIAddresses().includes((data.address || '').toLowerCase()) && 'GoodDollar UBI'
```

Comment:
this already returns the ubi addresses from all networks

Reply:
it was reverted